.github/workflows/test.yml

```diff
@@ -6,30 +6,54 @@ on:
   pull_request:
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
-  unix-test:
+  test:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest]
+        os:
+          - ubuntu-latest
+          # TODO: Test other OSs!
+          #- macos-latest
+          #- windows-latest
         emacs-version:
           - 27.2
-          - 28.1
-          - 29.3
+          - 28.2
           - 29.4
+        experimental: [false]
+        include:
+        - os: ubuntu-latest
+          emacs-version: snapshot
+          experimental: true
+        #- os: macos-latest
+        #  emacs-version: snapshot
+        #  experimental: true
+        #- os: windows-latest
+        #  emacs-version: snapshot
+        #  experimental: true
+        exclude:
+          - os: macos-latest
```

Comment:
I think this can be removed since we don't test on macos anyway ?

Reply:
I was planning to add the macos test in the future. But I guess we can remove it for now. :)